lib/src/components/NxDropdownIconOnly/NxDropdownIconOnly.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent, ReactElement } from 'react';
+import classnames from 'classnames';
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+
+import { Props, propTypes } from './types';
+import NxButton from '../NxButton/NxButton';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import './NxDropdownIconOnly.scss';
+import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
+import NxDropdownMenu from '../NxDropdownMenu/NxDropdownMenu';
+import useDropdownEvents from '../../util/useDropdownEvents';
+
+const NxDropdownIconOnly: FunctionComponent<Props> = function NxDropdownIconOnly(props) {
```

Comment:
There is a lot of duplicated with `NxDropdown` here.  The only distinction between the two is the JSX of the `toggle`.  That leads me to think there should be an abstract component that they both rely on, similar to how `AbstractTreeViewSelect` implemenents the common functionality between `NxTreeViewMultiSelect` and `NxTreeViewRadioSelect`.

Reply:
> There is a lot of duplicated with `NxDropdown` here. The only distinction between the two is the JSX of the `toggle`. That leads me to think there should be an abstract component that they both rely on, similar to how `AbstractTreeViewSelect` implemenents the common functionality between `NxTreeViewMultiSelect` and `NxTreeViewRadioSelect`.

That does make some sense, and could be used by the segmented button and any new components that might have a dropdown. However I don't really have any idea how to do this and suggest that given the scope that it would be best handled as a separate story.

Reply:
> > There is a lot of duplicated with `NxDropdown` here. The only distinction between the two is the JSX of the `toggle`. That leads me to think there should be an abstract component that they both rely on, similar to how `AbstractTreeViewSelect` implemenents the common functionality between `NxTreeViewMultiSelect` and `NxTreeViewRadioSelect`.
> 
> That does make some sense, and could be used by the segmented button and any new components that might have a dropdown. However I don't really have any idea how to do this and suggest that given the scope that it would be best handled as a separate story.

https://issues.sonatype.org/browse/RSC-674